lib/elixir/lib/path.ex

```diff
@@ -39,7 +39,7 @@ defmodule Path do
   """
   @spec absname(t) :: binary
   def absname(path) do
-    absname(path, System.cwd!())
+    absname(path, File.cwd!())
```

Comment:
`absname/1` and `expand/1,2` could potentially raise a different error than previous Elixir versions. I guess we are ok, but just to double check.